common/country_definitions/dvg_ameriga.txt

```diff
@@ -1,4 +1,4 @@
-﻿TWT = { #Tawantinsuyu
```

Comment:
File encoding issue:
File 'common/country_definitions/dvg_ameriga.txt' should be in utf8-bom encoding (will try to use it anyways)
File 'common/history/buildings/07_south_america.txt' should be in utf8-bom encoding (will try to use it anyways)

Reply:
this is what happens when I don't work on my main computer. It's clear now though.